SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -410,16 +412,24 @@ Plot {
 			if(ycoord.at(0).isSequenceableCollection) { // multi channel expansion when superposed
 				ycoord.flop.do { |y, i|
 					var mode = plotMode.wrapAt(i);
+					var xcoord = this.domainCoordinates(ycoord.size, mode);
 					Pen.beginPath;
 					Pen.fillColor = plotColor.wrapAt(i);
 					Pen.strokeColor = plotColor.wrapAt(i);
 					this.perform(mode, xcoord, y);
 				}
 			} {
+				var mode = plotMode.at(0);
+				var xcoord = this.domainCoordinates(ycoord.size, mode);
 				Pen.beginPath;
 				Pen.strokeColor = plotColor.at(0);
 				Pen.fillColor = plotColor.at(0);
 				this.perform(plotMode.at(0), xcoord, ycoord);
+				if (this.needsPenFill(mode)) {
```

Comment:
In the current design, the fill / stroke differece is handles in the plotMode itself. So `fill` or `stroke` should have been called already in `this.perform(plotMode.at(0), xcoord, ycoord);`.

Is there a reason this is needed?